<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>Reinhold Kainhofer &lt;reinhold@kainhofer.com&gt;</author>
 <comment>Configuration page for the print week mode.</comment>
 <class>CalPrintWeekConfig_Base</class>
 <widget class="QWidget" name="CalPrintWeekConfig_Base">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>684</width>
    <height>615</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Sans Serif'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Print week options:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="mDateRangeGroup">
     <property name="title">
      <string>Date &amp;&amp; Time Range</string>
     </property>
     <layout class="QGridLayout">
      <property name="leftMargin">
       <number>9</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>9</number>
      </property>
      <property name="bottomMargin">
       <number>9</number>
      </property>
      <property name="spacing">
       <number>6</number>
      </property>
      <item row="0" column="4">
       <spacer>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Expanding</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>16</width>
          <height>21</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="mFromDateLabel">
        <property name="toolTip">
         <string>Starting date for the print</string>
        </property>
        <property name="whatsThis">
         <string>Here you can choose which events should be printed based on their date. This check enables you to enter the start date of the date range. Use the &lt;i&gt;End date&lt;/i&gt; to enter the end date of the daterange.</string>
        </property>
        <property name="text">
         <string>&amp;Start date:</string>
        </property>
        <property name="buddy">
         <cstring>mFromDate</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="KDateComboBox" name="mFromDate">
        <property name="focusPolicy">
         <enum>Qt::StrongFocus</enum>
        </property>
        <property name="toolTip">
         <string>Starting date for the print</string>
        </property>
        <property name="whatsThis">
         <string>Here you can choose which events should be printed based on their date. This check enables you to enter the start date of the date range. Use the &lt;i&gt;End date&lt;/i&gt; to enter the end date of the daterange.</string>
        </property>
        <item>
         <property name="text">
          <string>2009-01-19</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLabel" name="mToTimeLabel">
        <property name="whatsThis">
         <string>All events which start later than the given time will not be printed.</string>
        </property>
        <property name="text">
         <string>End ti&amp;me:</string>
        </property>
        <property name="buddy">
         <cstring>mToTime</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="4">
       <spacer>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Expanding</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>110</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="3">
       <widget class="QTimeEdit" name="mToTime">
        <property name="whatsThis">
         <string>All events which start later than the given time will not be printed.</string>
        </property>
        <property name="time">
         <time>
          <hour>18</hour>
          <minute>0</minute>
          <second>0</second>
         </time>
        </property>
        <property name="displayFormat">
         <string>hh:mm</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="mFromTimeLabel">
        <property name="whatsThis">
         <string>All events which start earlier than the given time will not be printed.</string>
        </property>
        <property name="text">
         <string>Start &amp;time:</string>
        </property>
        <property name="buddy">
         <cstring>mFromTime</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QTimeEdit" name="mFromTime">
        <property name="whatsThis">
         <string>All events which start earlier than the given time will not be printed.</string>
        </property>
        <property name="time">
         <time>
          <hour>8</hour>
          <minute>0</minute>
          <second>0</second>
         </time>
        </property>
        <property name="displayFormat">
         <string>hh:mm</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="KDateComboBox" name="mToDate">
        <property name="focusPolicy">
         <enum>Qt::StrongFocus</enum>
        </property>
        <property name="whatsThis">
         <string>Here you can choose which events should be printed based on their date. This check enables you to enter the end date of the date range. Use the &lt;i&gt;Start date&lt;/i&gt; to enter the start date of the daterange.</string>
        </property>
        <item>
         <property name="text">
          <string>2009-01-19</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="mToDateLabel">
        <property name="whatsThis">
         <string>Here you can choose which events should be printed based on their date. This check enables you to enter the end date of the date range. Use the &lt;i&gt;Start date&lt;/i&gt; to enter the start date of the daterange.</string>
        </property>
        <property name="text">
         <string>&amp;End date:</string>
        </property>
        <property name="buddy">
         <cstring>mToDate</cstring>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="mPrintType">
     <property name="title">
      <string>Print Layout</string>
     </property>
     <layout class="QVBoxLayout">
      <property name="spacing">
       <number>6</number>
      </property>
      <property name="leftMargin">
       <number>9</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>9</number>
      </property>
      <property name="bottomMargin">
       <number>9</number>
      </property>
      <item>
       <widget class="QRadioButton" name="mPrintTypeFilofax">
        <property name="whatsThis">
         <string>The Filofax view prints one week per page, so all days have a large surface.</string>
        </property>
        <property name="text">
         <string>Print as &amp;Filofax page</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="mPrintTypeTimetable">
        <property name="whatsThis">
         <string>This view is similar to the weekview in KOrganizer. The week is printed in landscape layout. You can even use the same colors for the items if you check &lt;i&gt;Use Colors&lt;/i&gt;.</string>
        </property>
        <property name="text">
         <string>Print as &amp;timetable view</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="mPrintTypeSplitWeek">
        <property name="whatsThis">
         <string>This view is similar to the week view in KOrganizer. The only difference with the timetable view is the page layout. Timetables are printed in landscape, the split week view in portrait.</string>
        </property>
        <property name="text">
         <string>Print as split week view</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="mSecurity">
     <property name="title">
      <string>Security Exclusions</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="topMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QCheckBox" name="mExcludeConfidential">
        <property name="whatsThis">
         <string>Check this option to exclude confidential events.</string>
        </property>
        <property name="text">
         <string>Exclude c&amp;onfidential</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="mExcludePrivate">
        <property name="whatsThis">
         <string>Check this option to exclude private events</string>
        </property>
        <property name="text">
         <string>Exclude pri&amp;vate</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="mIncludeInfoGroup">
     <property name="title">
      <string>Include Information</string>
     </property>
     <layout class="QGridLayout" name="includeInfoLayout">
      <item row="0" column="0">
       <widget class="QCheckBox" name="mIncludeDescription">
        <property name="text">
         <string>Include &amp;descriptions</string>
        </property>
        <property name="toolTip">
         <string>Print item descriptions</string>
        </property>
        <property name="whatsThis">
         <string>Check this option if you want to see the item descriptions printed.</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QCheckBox" name="mIncludeTodos">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="whatsThis">
         <string>Check this option if you want to have to-dos on the print, placed by their due date.</string>
        </property>
        <property name="text">
         <string>Include to-&amp;dos that are due on the printed day(s)</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QCheckBox" name="mExcludeTime">
        <property name="whatsThis">
         <string>Check this option to exclude the time in description box</string>
        </property>
        <property name="text">
         <string>Exclude t&amp;ime from timetable detail display items</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="mGeneralGroup">
     <property name="title">
      <string comment="@title general print settings">General</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QCheckBox" name="mSingleLineLimit">
        <property name="toolTip">
         <string>Print items on one line</string>
        </property>
        <property name="whatsThis">
         <string>Check this option to limit events to a single line, truncating as necessary to save space.</string>
        </property>
        <property name="text">
         <string>Limit events in each day to a &amp;single line</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="mShowNoteLines">
        <property name="whatsThis">
         <string>Check this option to draw note lines.</string>
        </property>
        <property name="text">
         <string>Show note &amp;lines in Filofax layout</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="mColors">
        <property name="toolTip">
         <string>Print in color</string>
        </property>
        <property name="whatsThis">
         <string>The timetable view supports colors. If you want to make use of colors you should check this option. The category colors will be used.</string>
        </property>
        <property name="text">
         <string>&amp;Use colors</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="mPrintFooter">
        <property name="text">
         <string>Print &amp;Footer</string>
        </property>
        <property name="toolTip">
         <string>Print a datetime footer on each page</string>
        </property>
        <property name="whatsThis">
         <string>Check this box if you want to print a small footer on each page that contains the date of the print.</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>53</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KDateComboBox</class>
   <extends>KComboBox</extends>
   <header>kdatecombobox.h</header>
  </customwidget>
  <customwidget>
   <class>KComboBox</class>
   <extends>QComboBox</extends>
   <header>kcombobox.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>mFromDate</tabstop>
  <tabstop>mFromTime</tabstop>
  <tabstop>mToDate</tabstop>
  <tabstop>mToTime</tabstop>
  <tabstop>mPrintTypeFilofax</tabstop>
  <tabstop>mPrintTypeTimetable</tabstop>
  <tabstop>mPrintTypeSplitWeek</tabstop>
  <tabstop>mIncludeTodos</tabstop>
  <tabstop>mShowNoteLines</tabstop>
  <tabstop>mSingleLineLimit</tabstop>
  <tabstop>mColors</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
